### PR TITLE
ECO 1568/Update Portuguese

### PR DESCRIPTION
**Pull request by AliAsadi**
#### Main purpose:
Supporting Different Language (Portuguese)

#### Technical description:
* Add Portuguese support to backup & restore modules
* Update marketplace texts
* Add missing Portuguese strings
* Fix design issue

### CODE REVIEW

File: recovery/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    <string name="kinrecovery_restore">Restaurar</string>
+    <string name="kinrecovery_backup">Backup</string>
+
+    <!--Backup-->
+    <string name="kinrecovery_keep_your_kin_safe">Mantenha seu Kin seguro</string>
+    <string name="kinrecovery_backup_description">Fazendo o backup do seu saldo atual irá lhe permitir\n recuperá-lo, caso você troque o telefone, apague o app\n ou seus dados.</string>
+    <string name="kinrecovery_let_s_go">Vamos lá</string>
+    <string name="kinrecovery_create_password">Criar uma Senha</string>
+    <string name="kinrecovery_create_password_description">Para fazer o backup do seu Kin.\n primeiro crie uma senha forte. Ela deve ter:\n (9 caracters, 1 maiúscula, 1 caracter especial (#, !, @) e 1 número)</string>
```
Comment (OrenZak) [inline recovery/src/main/res/values-pt-rBR/strings.xml]: it seems like after `\n` you have space is it intended?

File: recovery/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <string name="kinrecovery_restore">Restore</string>
     <string name="kinrecovery_backup">Backup</string>
 
-    <string name="kinrecovery_steps_format">%d/%d</string>
+    <string name="kinrecovery_steps_format" translatable="false">%d/%d</string>
```
Comment (OrenZak) [inline recovery/src/main/res/values/strings.xml:7]: why the `translatable=false` is necessary.

**Review by OrenZak — APPROVED** [review]
Just see my comments 💯

File: recovery/src/main/res/values-pt-rBR/strings.xml (hunk 1 above)
Comment (AliAsadi) [reply recovery/src/main/res/values-pt-rBR/strings.xml]: no, updated.

File: recovery/src/main/res/values/strings.xml (hunk 2 above)
Comment (AliAsadi) [reply recovery/src/main/res/values/strings.xml:7]: To remove the warning message from the IDE, they ask to translate this line to Portuguese.

**Pull request merged**